src/react/projects/spark-core-react/package.json

```diff
@@ -35,6 +35,8 @@
     "lodash": "^4.17.11",
     "prop-types": "^15.6.2",
     "react-animate-height": "^2.0.7",
-    "tiny-date-picker": "^3.2.8"
+    "react-router-dom": "^4.3.1",
```

Comment:
do we need react router as a dep for spark-react?

Reply:
it was for testing

![image](https://user-images.githubusercontent.com/4342363/54227373-2a026180-44d6-11e9-8d05-7e5f0b903fda.png)

Reply:
![image](https://user-images.githubusercontent.com/4342363/54227396-325a9c80-44d6-11e9-9fcb-1e62ea1883e2.png)

Reply:
otherwise "Link" will not work

Reply:
working on this now, since we're doing the TagName strategy, we probably won't need this anymore.

Reply:
@RVMendoza can you remove this dependency from Spark Core React, looks like its not needed anymore? 
react-router-dom